dockerfiles/mlrun-api/Dockerfile

```diff
@@ -51,14 +51,28 @@ RUN python -m pip install --upgrade pip~=${MLRUN_PIP_VERSION}
 
 WORKDIR /mlrun
 
-COPY ./dockerfiles/mlrun-api/requirements.txt ./mlrun-api-requirements.txt
+COPY ./dockerfiles/mlrun-api/requirements.txt dockerfiles/mlrun-api/requirements.txt
 COPY ./extras-requirements.txt ./extras-requirements.txt
 COPY ./requirements.txt ./
+COPY ./dockerfiles ./dockerfiles
+COPY ./pipeline-adapters ./pipeline-adapters
+COPY ./requirements.txt  .
+COPY ./dev-requirements.txt  .
+COPY ./conda-arm64-requirements.txt  .
+COPY ./packages.py .
+COPY ./dependencies.py  .
+COPY ./setup.py .
```

Comment:
Don't you need the mlrun client code to install it? (e.g. `COPY ./mlrun .`)

Reply:
We are not installing the mlrun client code, just the dependencies.